app/src/main/java/org/cosmicide/rewrite/extension/editor.kt

```diff
@@ -24,6 +24,6 @@ fun CodeEditor.setLanguageTheme(language: EditorLanguage) {
  * Sets the font and enables highlighting of the current line for the code editor.
  */
 fun CodeEditor.setFont() {
-    typefaceText = ResourcesCompat.getCachedFont(context, R.font.noto_sans_mono)
+    typefaceText = ResourcesCompat.getFont(context, R.font.noto_sans_mono)
```

Comment:
getCachedFont would be better here i think

Reply:
With cachedfont, the font is not loaded.

Reply:
With cachedfont, the font is not loaded.

Reply:
It checks if its cached and sets it. atleast in emulator, it loads properly

Reply:
You check? With the `getCachedFont()` method on my device, the font just didn't load.

Reply:
it did work

Reply:
Check last artifact (not pr's artifact)